internal/engine/experiment/stunreachability/stunreachability.go

```diff
@@ -20,7 +22,7 @@ import (
 
 const (
 	testName    = "stunreachability"
-	testVersion = "0.2.0"
+	testVersion = "0.3.0"
```

Comment:
Bumped version because we're changing the input type (now it's an URL so that we have only URLs in the URLs table maintained by `ooniprobe`) and we're removing the default input from the experiment implementation (now the default input comes from the input loader, which also helps with `ooniprobe`).

The companion `ooni/spec` PR already landed at https://github.com/ooni/spec/pull/227